pkg/controller/awsfederatedaccountaccess/awsfederatedaccountaccess_controller.go

```diff
@@ -159,19 +159,13 @@ func (r *ReconcileAWSFederatedAccountAccess) Reconcile(request reconcile.Request
 	}
 
 	// Check if the FAA has the uid label
-	if !hasUIDLabel(currentFAA) {
+	if !hasLabel(currentFAA, "uid") {
```

Comment:
Can we make the `"uid"` a var in `shared_types.go` for v1alpha1?

Reply:
:+1: